include/multipass/vm_image_info.h

```diff
@@ -40,5 +40,13 @@ class VMImageInfo
     int64_t size;
     bool verify;
 };
+
```

Comment:
C++20 can do default three way comparison `auto operator<=>(const VMImageInfo& other) const = default;` to save us from the boilerplate thing. However, since we are not there yet, this is the best we can do.

Reply:
Yup, exactly.  And there are a few other classes that use `std::tie()` as well.